openedx/core/djangoapps/user_authn/cookies.py

```diff
@@ -182,7 +182,7 @@ def _set_deprecated_user_info_cookie(response, request, user, cookie_settings):
     """
     user_info = _get_user_info_cookie_data(request, user)
     response.set_cookie(
-        settings.EDXMKTG_USER_INFO_COOKIE_NAME.encode('utf-8'),
+        six.text_type(settings.EDXMKTG_USER_INFO_COOKIE_NAME.encode('utf-8')),
```

Comment:
So this was previously providing bytes but with this change, it will provide unicode?  Why not just remove the call to `encode`?  What's the upstream error?

Reply:
My bad. removed.